cmake/projects/chaiscript/hunter.cmake

```diff
@@ -0,0 +1,30 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  chaiscript
+  VERSION
+  "6.0.0"
+  URL
+  https://github.com/hunter-packages/ChaiScript/archive/hunter.zip
```

Comment:
Please use stable release.

Reply:
I see, I grabbed the wrong URL.  Thank you.